oe_theme.theme

```diff
@@ -1315,3 +1315,106 @@ function _oe_theme_trim_textfield($text, int $size = 150) {
   // Return the unchanged $text since we don't support other scenarios.
   return $text;
 }
+
+/**
+ * Implements hook_preprocess_paragraph().
+ */
+function oe_theme_preprocess_paragraph__oe_facts_figures(array &$variables): void {
+  $paragraph = $variables['paragraph'];
+  $variables['title'] = $paragraph->get('field_oe_title')->value;
+  $variables['view_all'][0]['path'] = $paragraph->get('field_oe_link')->first()->getUrl();
+  $variables['view_all'][0]['label'] = $paragraph->get('field_oe_link')->first()->get('title')->getValue();
+  $variables['items'] = [];
+
+  foreach ($paragraph->get('field_oe_paragraphs')->referencedEntities() as $sub_paragraph) {
+    /** @var \Drupal\paragraphs\Entity\Paragraph $sub_paragraph */
+    $variables['items'][] = [
+      'icon' => _oe_theme_icons_convert($sub_paragraph->get('field_oe_icon')->value),
```

Comment:
See below, pass the raw value here and use the new Twig filter to convert icons into ECL icon names.